### DISCUSSION TITLE
Simulating a much larger (imaginary) GPU

### DISCUSSION THREAD

**Discussion by rvnonsec** [accel-sim/accel-sim-framework#346] [Q&A]
Hi,

Can someone help with the considerations to extend the config to represent a much larger machine than ampere (or even blackwell). For eg, say 256 to 512SMs and scaling up cache/memory bw similarly?

Specifically, outside of obvious things like SM count and memory, what other config knobs are likely to have to be reworked for such a config 

Thanks!

**Comment by JRPan** [comment]
We have created a config for A100. However, please note it has not been tested yet. Some Ampere-specific features are under dev. 
Both trace and execution mode uses the same backend. However, some API calls/instructions may not be available in the PTX execution-driven mode.

**Comment by JRPan** [comment]
https://github.com/accel-sim/accel-sim-framework/pull/344
https://github.com/accel-sim/gpgpu-sim_distribution/pull/80
